src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'clear') {
```

Comment:
use switch case

Reply:
Why is the switch case better?

Reply:
cause when you have one value to compare it's more declarative to use switch case